CODEOWNERS

```diff
@@ -42,6 +42,7 @@ go.mod @fleetdm/go
 
 # Expanded table documentation
 /schema @eashaw
+/schema/tables @rachaelshaw
```

Comment:
The standard questions:
- is there other places in the handbook that might mention the previous codeowner that we need to change?  (TODO: do a search for "rachael" "schema"
- what about website/config/custom.js?


Extra question:
- why only /schema/tables and not /schema?  Seems easier to have it be one thing.  You can add a comment off to the side to explain why, or consolidate